client/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Logo from '../Logo/Logo.js';
+import styles from './Navbar.module.scss';
+import { Link } from 'react-router-dom';
+import SearchInput from '../SearchInput/SearchInput.js';
+import NavLinks from '../NavLinks/NavLinks.js';
+
+const Navbar = () => {
```

Comment:
i guess you don'y need the curly brackets and the return if you're only returning HTML.
```suggestion
const Navbar = () => (
```